### ISSUE TITLE
Support opening application from the path editbox

### ISSUE THREAD

**Issue by mpatel8** [files-community/Files#5677]
### Description

Similar to Windows Explorer, ability to launch a command window and execute the command typed into the navigation bar, including locating the command in the path.

### Expected behavior

Example: H:\commands is  in my PATH variable. AT that location there is a bat file m.bat. In windows explorer I can type in m in the navigation bar to open a command window which then runs H:\commands\m.bat as that is the only m.bat or m.exe in only PATH locations.

### Files Version

v1.5.0.0

### Windows Version

Version 10.0.18363.1679

**Comment by StefanLobbenmeier** [comment]
I am here from #7814 

I determined that the location to add this feature is most likely here: https://github.com/files-community/Files/blob/afb77d9f6399066af55b035a4843a63840186074/src/Files/ViewModels/NavToolbarViewModel.cs#L956

I tried adding the following code:

```cs
        private static void LaunchApplicationFromPath(string currentInput, string workingDir)
        {
            var trimmedInput= currentInput.Trim();
            var fileName = trimmedInput;
            var arguments = "";
            if (trimmedInput.Contains(' '))
            {
                var positionOfBlank = trimmedInput.IndexOf(' ');
                fileName = trimmedInput.Substring(0, positionOfBlank);
                arguments = currentInput.Substring(currentInput.IndexOf(' '));
            }
            
            var psi = new ProcessStartInfo
            {
                FileName = fileName,
                Arguments = arguments,
                CreateNoWindow = false,
                UseShellExecute = true,
                WindowStyle = ProcessWindowStyle.Normal,
                WorkingDirectory = workingDir
            };
            Process.Start(psi);
        }
```

But this seems to crash the application. I instead tried 


  ```cs
            var connection = await AppServiceConnectionHelper.Instance;
            if (connection != null)
            {
                var value = new ValueSet()
                {
                    { "Arguments", "LaunchApp" },
                    { "WorkingDirectory", workingDir },
                    { "Application", fileName },
                    { "Parameters", arguments }
                };
                await connection.SendMessageAsync(value);
            }
```

But in my locally built dev build the FilesFullTrust did not start. Will try with the CI build now.

**Comment by d2dyno1** [comment]
This should be resolved with #7814 @mpatel8
